### ISSUE TITLE
Investigate issue

### ISSUE THREAD

**Issue by danuker** [chevah/github-hooks-server#30]
As a consequence of [this PR review](https://github.com/chevah/proatria.com/pull/9#pullrequestreview-847920064), [the Azure logs](https://portal.azure.com/#@proatria.onmicrosoft.com/resource/subscriptions/a98aee2a-9755-4a01-a2bd-aaa61ea1d71d/resourceGroups/sls-weur-dev-githubhooks-rg/providers/Microsoft.Web/sites/sls-weur-dev-githubhooks/appInsightsQueryLogs) showed success, but the actions required were not performed.

See them [here](https://portal.azure.com/#@proatria.onmicrosoft.com/resource/subscriptions/a98aee2a-9755-4a01-a2bd-aaa61ea1d71d/resourceGroups/sls-weur-dev-githubhooks-rg/providers/Microsoft.Web/sites/sls-weur-dev-githubhooks/appInsightsQueryLogs) with this query:

```
traces
| order by timestamp
| where timestamp > todatetime('2022-01-10 14:52')
| where timestamp < todatetime('2022-01-10 14:54')
| project timestamp, message, severityLevel
```

Try to reproduce locally, in order to determine what was the problem.

**Comment by danuker** [comment]
My best explanation so far:

1. Setting Adi as the assignee failed (permission error) when Adi submitted the approving reviews. This led to Cristina staying as the assignee.
2. I assigned Adi manually to the ticket, wrongly expecting the system to look at the reviews so far. This does not happen yet, is due in #27 .
3. The hook did not then react to my approvals. I then realized it was because Adi was assigned, so it still waited for a review from Adi.
4. I manually unassigned Adi from the ticket, assigned myself, and indeed, the system correctly assigned Cristina again, proving the permissions now work on `chevah/proatria.com`.

Closing this bug, as the root cause was fixed (permissions). The other contributing factor will be fixed in #27.
